src/cmdc/datasets/uscensus/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP SCHEMA IF EXISTS uscensus CASCADE;
+
+CREATE SCHEMA uscensus;
+
+/* Create and define metadata tables */
+CREATE TABLE uscensus.acs_variables (
+    "id" SERIAL UNIQUE,
+    "year" INTEGER,
```

Comment:
I recall discussing this breifly, but can you remind my why the year goes in the variables table instead of the data one?

Reply:
An element of `id` in the `acs_variables` column is associated with a `census_id` in either `acs1`/`acs5` for a particular `year`. It doesn't happen often (tbh, I don't even know if I've even found an example), but, hypothetically, a particular `census_id` could be defined differently in different years.

In the view that users would use, we'll definitely expose the year the variable was collected though.